src/WattleScript.Interpreter/DataTypes/DynValue.cs

```diff
@@ -693,20 +707,17 @@ internal double AssertNumber(int stage)
 		public bool TryCastToNumber(out double d)
 		{
 			ref DynValue rv = ref ScalarReference(ref this);
-			if (rv.Type == DataType.Number)
-			{
-				d = rv.Number;
-				return true;
-			}
-			else if (rv.Type == DataType.String)
+			switch (rv.Type)
```

Comment:
Unnecessary change for this PR, remove.

Reply:
Fixed in a8172ea1ce414989783dccc972befb225d8def58